examples/diffusion/python_stable_diffusion_21/README.md

```diff
@@ -21,7 +21,7 @@ python3 -m venv sd_venv
 Install dependencies
 
 ```bash
-pip install -r requirements.txt
+pip install -r torch_requirements.txt -r requirements.txt
```

Comment:
Same thing as above. if possible, leave this as `pip install -r requirements.txt`